package.json

```diff
@@ -54,6 +54,9 @@
     "wait-on": "7.2.0"
   },
   "pnpm": {
+    "overrides": {
+      "eslint-config-galex>typescript": "$typescript"
+    },
```

Comment:
ESLint complains about the new `module: 'preserve'` setting in `tsconfig`. Why now that I'm updating `h5wasm` and not when I actually switched to that setting and upgraded typescript, I have no idea :shrug: